Source/Shaders/ModelExperimental/FeatureStageVS.glsl

```diff
@@ -0,0 +1,35 @@
+vec2 computeSt(float featureId)
+{
+    float stepX = model_textureStep.x;
+    float centerX = model_textureStep.y;
+
+    #ifdef MULTILINE_BATCH_TEXTURE
+    float stepY = model_textureStep.z;
+    float centerY = model_textureStep.w;
+
+    float xId = mod(featureId, model_textureDimensions.x); 
+    float yId = floor(featureId / model_textureDimensions.x);
+    
+    return vec2(centerX + (xId * stepX), centerY + (yId * stepY));
+    #else
+    return vec2(centerX + (featureId * stepX), 0.5);
+    #endif
+}
+
+vec3 featureStage(vec3 position)
+{
+    float featureId;
+
+    #ifdef FEATURE_ID_TEXTURE
+    featureId = floor(texture2D(FEATURE_ID_TEXTURE, FEATURE_ID_TEXCOORD).FEATURE_ID_CHANNEL * 255.0 + 0.5);
```

Comment:
I know we're not supporting multiple feature tables yet, though when we do, is the intent that you only use one at a time?

Reply:
I think the right progression of functionality is:

1. A single attribution method is used.
2. A single attribution method is used but the user gets to decide index of feature ID attribute or tetxure
3. Multiple attribution methods are supported simultaneously (@lilleyse would have to confirm if this is possible)

This PR adds support for the first type on this list.

Reply:
For custom shaders and GPU styling it should be possible to use properties from all the feature tables, with some special handling if there are name collisions.

For picking it's a little bit harder and I think 2 is the right compromise